File: IUST 4th Sem/Data Structure/Find the Product of Elements of Array.cpp
#include <iostream>
using namespace std;

void scan_array(int arr[], int n)
{
    cout<< "Enter the Elements of array : ";
    for(int i=0; i<n; i++){
        cin >> arr[i];
    }
}

int product_array(int arr[], int n)
{
    int prod = 1;
    for(int i=0; i<n; i++){
        prod *= arr[i];
    }
    return prod;
}

int main() {
    int arr[100];
    int num, prod;;
    
    cout << "Enter the number of elements of Array : ";
    cin >> num;
    
    scan_array(arr, num);
    prod = product_array(arr, num);
    
    cout << "Product of Elements of Array = " << prod;

    return 0;
}

File: IUST 4th Sem/Data Structure in Python/Find the product of elements of Array.py
def product_array(arr, n):
    pro = 1
    for i in range(len(arr)):
        pro *= arr[i]
    return pro

arr = [1, 2, 3]
s = len(arr)
p = product_array(arr, s)
print("Product of Elements of Array = ", p)
